docs/team/r-ramana.md

```diff
@@ -39,14 +50,23 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
     - `getIsArchived`
     - `getPinned`
 1. Added a couple of `formatNotes` methods into the `Formatter` class.
+
+### Enhancements to existing features
+1. Added ASCII art to make the application more user-friendly.
 1. Refactored code to abstract out the successful/unsuccessful execution messages from the individual command classes to its own `CommandMessage` class.
-1. Added ASCII art to make the application more user-friendly
 
 ### Code contributed
+[Reposense Link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=r-ramana&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
 
+### Contributions to Team Based Task
+1. Setting up the GitHub team organization and repository.
+1. Maintained Issues and Reviewed PR alongside Jun Xiang.
+1. Updating of User/Developer documents that are not feature specific.
+1. Maintained Gantt Chart.
+1. Created a Development Practices Document.
 
 ### Contributions to User Guide
-Created the first draft of the Developer Guide and provided a template for the rest to follow. Also in charge of the overall formatting for the Developer Guide.
+Created the first draft of the User Guide and provided a template for the rest to follow. Also in charge of the overall formatting for the User Guide.
 Filled up the following:
 1. Introduction
```

Comment:
Can link it to the specific segment in the UG and DG

Reply:
Will do in the next round of changes